### DISCUSSION TITLE
Weblate complains about translations - "Things to check" infobox

### DISCUSSION THREAD

**Discussion by rikusen0335** [rob006-software/flarum-translations#568] [General]
Title is very abstract, so let me describe details.

## Trailing stop
When translate wny word or sentence, there are situations that using different "stop".

e.g.
![image](https://user-images.githubusercontent.com/19174234/97079721-431b4000-1631-11eb-86a4-7272f915e9fc.png)

How this can be solved or, should I force to use same "stop"?

## Unchanged translation
you might know there are arrow translations like this:
![image](https://user-images.githubusercontent.com/19174234/97079767-78c02900-1631-11eb-9f08-d3e4c17e887d.png)

Sometimes this is not correct. But there's way same word should be used.
How this can be solved or, should I force to translate them?

## XML markup
Uhhh, sorry I don't know what's happening. Could you tell me which is incorrect?
![image](https://user-images.githubusercontent.com/19174234/97079874-34815880-1632-11eb-9273-958cdf95958f.png)

---

Finally thank you for making the lots of translation in one place. This is kinda important to translating anything.

**Comment by rob006** [comment]
You mean these automated checks:
![8820fc79](https://user-images.githubusercontent.com/5972388/97080102-53afc400-15f9-11eb-9603-5b7b37166bc0.png)
?

Language pack maintainer should be able to dismiss them (@Littlegolden in this case), so you don't need to adjust translation if report does not make sense.

**Comment by rikusen0335** [comment]
Oh I did make sense! Thank you for answering.

**Comment by littlegolden** [comment]
Can you please add @rikusen0335 to the Japanese language maintainer?

**Comment by rob006** [comment]
@Littlegolden Done.

@rikusen0335 You may want to start marking verified translations as approved. Right now any registered user can make translations, so this is main tool to make sure that there are no invalid translations submitted by someone.

**Comment by littlegolden** [comment]
thanks~

**Comment by rikusen0335** [comment]
Thanks. I will take care to verify anything for now.
